manifests/cluster-authentication-operator_02_config.crd.yaml

```diff
@@ -12,3 +12,96 @@ spec:
     singular: authentication
   subresources:
     status: {}
```

Comment:
I'm not sure about this subresources field.  Looks like it's omitted from cluster-network-operator crd, we might want to omit that here?

Reply:
NOPE, that belongs, nevermind